apps/js-sdk/firecrawl/src/index.ts

```diff
@@ -947,10 +947,23 @@ export class CrawlWatcher extends TypedEventTarget<CrawlWatcherEvents> {
   public status: CrawlStatusResponse["status"];
   public id: string;
 
+  private static loadWebSocket() {
+    try {
+      return require('isows').WebSocket;
+    } catch {
+      try {
+        return import('isows').then(m => m.WebSocket);
```

Comment:
Missing an await here, right? Either here or on line 964.

Reply:
Thanks for catching that! I've updated the code to properly handle the WebSocket loading. Could you please review the changes again?

Reply:
This can cause a race condition. I'd prefer importing it lazily in an already-asynchronous function, like the watch crawl endpoint's function.